src/opamBaseParser.mly

```diff
@@ -264,7 +250,7 @@ let main lexer lexbuf file_name =
           lexer lexbuf
   in
   let result =
-    try with_clear_parser (main lexer lexbuf) file_name
```

Comment:
no menhir doesn't use the internal states from `Stdlib.Parsing` so it would be pointless to call this function, and as far as i know it doesn't have an equivalent function.